src/router-configuration.ts

```diff
@@ -16,11 +18,14 @@ export class RouterConfiguration {
     hashChange?: boolean;
     silent?: boolean;
   } = {};
-  pipelineSteps: Array<{name: string, step: Function|PipelineStep}> = [];
+  pipelineSteps: Array<{ name: string, step: Function | PipelineStep }> = [];
   title: string;
   titleSeparator: string;
-  unknownRouteConfig: string|RouteConfig|((instruction: NavigationInstruction) => string|RouteConfig|Promise<string|RouteConfig>);
-  viewPortDefaults: {[name: string]: {moduleId: string|void; [key: string]: any}};
+  unknownRouteConfig: string | RouteConfig | ((instruction: NavigationInstruction) => string | RouteConfig | Promise<string | RouteConfig>);
+  viewPortDefaults: { [name: string]: { moduleId: string | void;[key: string]: any } };
```

Comment:
nitpick but you lost the space after the semicolon, maybe break this out onto multi lines